src/main/cli/cli.c

```diff
@@ -2817,8 +2815,13 @@ static void cliVtxTable(char *cmdline)
         uint16_t bandfreq[VTX_TABLE_MAX_CHANNELS];
         int channel = 0;
         int channels = vtxTableConfigMutable()->channels;
+        bool isFactory = false;
 
         for (channel = 0; channel <  channels && (tok  = strtok_r(NULL, " ", &saveptr)); channel++) {
+            if (channel == 0 && !isdigit(tok[0])) {
```

Comment:
We can as well do a proper string comparison here, and throw an error if there is no match for `[FACTORY|USER]`.

Reply:
Makes sense, ill change it.
I have actually been thinking about the choice of `USER`. Maybe `CUSTOM` would fit better? Not that it makes much of a difference either way.

Reply:
Yeah, `CUSTOM` makes more sense.